frontend/test/__support__/cypress.js

```diff
@@ -203,55 +203,39 @@ export function createBasicAlert({ firstAlert, includeNormal } = {}) {
 /*****************************************
  **            QA DATABASES             **
  ******************************************/
-export function addMongoDatabase(db_display_name = "QA Mongo4") {
+
+export function addMongoDatabase(name = "QA Mongo4") {
   // https://hub.docker.com/layers/metabase/qa-databases/mongo-sample-4.0/images/sha256-3f568127248b6c6dba0b114b65dc3b3bf69bf4c804310eb57b4e3de6eda989cf
-  cy.log("**-- Adding Mongo 4 DB --**");
-  cy.request("POST", "/api/database", {
-    engine: "mongo",
-    name: db_display_name,
-    details: {
-      host: "localhost",
-      dbname: "sample",
-      port: 27017,
-      user: "metabase",
-      pass: "metasample123",
-      authdb: "admin",
-      "additional-options": null,
-      "use-srv": false,
-      "tunnel-enabled": false,
-    },
-    auto_run_queries: true,
-    is_full_sync: true,
-    schedules: {
-      cache_field_values: {
-        schedule_day: null,
-        schedule_frame: null,
-        schedule_hour: 0,
-        schedule_type: "daily",
-      },
-      metadata_sync: {
-        schedule_day: null,
-        schedule_frame: null,
-        schedule_hour: null,
-        schedule_type: "hourly",
-      },
-    },
-  });
+  addQADatabase("mongo", name, 27017);
+}
+
+export function addPostgresDatabase(name = "QA Postgres12") {
+  // https://hub.docker.com/layers/metabase/qa-databases/postgres-sample-12/images/sha256-80bbef27dc52552d6dc64b52796ba356d7541e7bba172740336d7b8a64859cf8
+  addQADatabase("postgres", name, 5432);
+}
+
+export function addMySQLDatabase(name = "QA MySQL8") {
+  // https://hub.docker.com/layers/metabase/qa-databases/mysql-sample-8/images/sha256-df67db50379ec59ac3a437b5205871f85ab519ce8d2cdc526e9313354d00f9d4
+  addQADatabase("mysql", name, 3306);
 }
 
-export function addPostgresDatabase(db_display_name = "QA Postgres12") {
-  cy.log("**-- Adding Postgres 12 DB --**");
+function addQADatabase(engine, db_display_name, port) {
+  const PASS_KEY = engine === "mongo" ? "pass" : "password";
+  const AUTH_DB = engine === "mongo" ? "admin" : null;
+  const OPTIONS = engine === "mysql" ? "allowPublicKeyRetrieval=true" : null;
```

Comment:
At some point, we may need a simple object to hold the different password/auth/options for each database. But this one is good for now.